integration-tests/test_cdk.py

```diff
@@ -5,22 +5,25 @@
 from validation import validate_all
 
 namespace = os.environ.get('TEST_CHARM_NAMESPACE', 'containers')
-charm_channel = os.environ.get('TEST_CHARM_CHANNEL', 'stable')
-snap_channel = os.environ.get('TEST_SNAP_CHANNEL', '1.9/stable')
+test_charm_channel = os.environ.get('TEST_CHARM_CHANNEL', 'stable')
+test_snap_channel = os.environ.get('TEST_SNAP_CHANNEL', '1.9/stable')
 test_cloud = os.environ.get('TEST_CLOUD', '')
 bundles_csv = os.environ.get('TEST_BUNDLES', default_bundles())
+upgrade_from_snap_channel = os.environ.get('UPGRADE_FROM_SNAP_CHANNEL', 'stable')
+upgrade_from_charm_channel = os.environ.get('UPGRADE_FROM_CHARM_CHANNEL', 'edge')
```

Comment:
So close!!! You want `stable` here too, because that's what we want when doing the initial deploy during test_upgrade.  Rather moot, i know, because this will be overridden by the job, but let's do it right anyway.